PowerEditor/src/winmain.cpp

```diff
@@ -451,17 +455,26 @@ int WINAPI wWinMain(HINSTANCE hInstance, HINSTANCE, PWSTR pCmdLine, int)
 	cmdLineParams._easterEggName = getEasterEggNameFromParam(params, cmdLineParams._quoteType);
 	cmdLineParams._ghostTypingSpeed = getGhostTypingSpeedFromParam(params);
 
+	generic_string pluginMessage;
+	if (getParamValFromString(FLAG_PLUGIN_MESSAGE, params, pluginMessage))
+	{
+		if (pluginMessage.length() >= 2)
+		{
+			if (pluginMessage.front() == '"' && pluginMessage.back() == '"')
+			{
+				pluginMessage = pluginMessage.substr(1, pluginMessage.length() - 2);
+			}
+		}
+		cmdLineParams._pluginMessage = pluginMessage;
+	}
+

```

Comment:
The scan for `-pluginMessage=` will have to precede that for `-p`, which follows on line 474 below. Otherwise, the scan for `-p` will consume the `-plugin=` param because of the smaller string match.